### PR TITLE
Bump vault/salt versions

### PR DESCRIPTION
**Pull request by dafyddj**
see commits

### CODE REVIEW

File: vault/defaults.yaml [hunk 1]
```diff
@@ -1,10 +1,9 @@
 vault:
-  version: 0.9.1
+  version: 0.9.6
```
Comment (javierbertoli) [inline vault/defaults.yaml]: Current version is [0.11.2](https://www.vaultproject.io/downloads.html), any reason to bump to `0.9.6`?

Comment (dafyddj) [reply vault/defaults.yaml]: vault v0.10 introduces kv engine v2 by default, which is not yet supported by salt

Comment (dafyddj) [reply vault/defaults.yaml]: Ok, I see what you mean.

**Comment by javierbertoli** [comment]
Thanks @dafyddj , and thanks for considering my comments too :smile:

**Comment by dafyddj** [comment]
👍@javierbertoli , your comments made sense in the wider scheme of things.

**Pull request merged**